tensorflow_addons/image/sparse_image_warp_test.py

```diff
@@ -26,75 +26,32 @@
 from tensorflow_addons.utils.resource_loader import get_path_to_datafile
 
 
-def test_zero_shift():
-    """Run assert_zero_shift for various hyperparameters."""
-    for order in (1, 2):
-        for regularization in (0, 0.01):
-            for num_boundary_points in (0, 1):
-                assert_zero_shift(order, regularization, num_boundary_points)
-
-
-def assert_zero_shift(order, regularization, num_boundary_points):
-    """Check that warping with zero displacements doesn't change the
-    image."""
-    batch_size = 1
-    image_height = 4
-    image_width = 4
-    channels = 3
-
-    image = np.random.uniform(size=[batch_size, image_height, image_width, channels])
-
-    input_image = tf.constant(np.float32(image))
-
-    control_point_locations = [[1.0, 1.0], [2.0, 2.0], [2.0, 1.0]]
-    control_point_locations = tf.constant(
-        np.float32(np.expand_dims(control_point_locations, 0))
-    )
-
-    control_point_displacements = np.zeros(control_point_locations.shape.as_list())
-    control_point_displacements = tf.constant(np.float32(control_point_displacements))
-
-    (warped_image, _) = sparse_image_warp(
-        input_image,
-        control_point_locations,
-        control_point_locations + control_point_displacements,
-        interpolation_order=order,
-        regularization_weight=regularization,
-        num_boundary_points=num_boundary_points,
-    )
+def test_get_boundary_locations():
+    image_height = 11
+    image_width = 11
+    num_points_per_edge = 4
+    locs = _get_boundary_locations(image_height, image_width, num_points_per_edge)
+    num_points = locs.shape[0]
+    np.testing.assert_equal(num_points, 4 + 4 * num_points_per_edge)
+    locs = [(locs[i, 0], locs[i, 1]) for i in range(num_points)]
+    for i in (0, image_height - 1):
+        for j in (0, image_width - 1):
+            assert (i, j) in locs, "{},{} not in the locations".format(i, j)
+
+        for i in (2, 4, 6, 8):
+            for j in (0, image_width - 1):
+                assert (i, j) in locs, "{},{} not in the locations".format(i, j)
```

Comment:
You can remove the error message. The one provided by pytest will have more info.

Reply:
Thanks @gabrieldemarmiesse.